checkov/terraform/checks/resource/base_resource_check.py

```diff
@@ -28,6 +29,21 @@ def __init__(
         self.supported_resources = supported_resources
         resource_registry.register(self)
 
+    @staticmethod
+    def _is_variable_dependant(value: Any) -> bool:
+        if not isinstance(value, str):
+            return False
+
+        if value.startswith('var.') or value.startswith('local.') or value.startswith('module.'):
```

Comment:
```suggestion
        if value.startswith(('var.', 'local.', 'module.')):
```
if my other comment is correct, then you can remove the `var.` case there